### PR TITLE
Updated naming conventions around domain names, part 7

### PR DESCRIPTION
**Pull request by orangejenny**
@dannyroberts

### CODE REVIEW

**Comment by orangejenny** [comment]
or @esoergel

File: corehq/apps/accounting/models.py [hunk 1]
```diff
@@ -904,16 +904,15 @@ def user_fee(self):
         if self.user_feature is not None:
             return "USD %d" % self.user_feature.per_excess_fee
 
-    def feature_charges_exist_for_domain(self, domain,
-                                         start_date=None, end_date=None):
-        domain = ensure_domain_instance(domain)
```
Comment (esoergel) [inline corehq/apps/accounting/models.py]: yikes

**Review by esoergel — APPROVED** [review]
:heart:

File: corehq/apps/accounting/models.py (hunk 1 above)
Comment (orangejenny) [reply corehq/apps/accounting/models.py]: `ensure_domain_instance`? It's kind of interesting how our code varies by app. Accounting tends to making `self.domain` an object - and uses `ensure_domain_instance` liberally - whereas in most of the rest of code I looked at, `self.domain` is a domain name.

Comment (esoergel) [reply corehq/apps/accounting/models.py]: I was "yikes"ing that here `domain` refers to both the string and the object in the same method.  We do somewhat frequently do the pattern of accepting `object_or_id` and then converting to object - just kinda ugly.

Comment (orangejenny) [reply corehq/apps/accounting/models.py]: aha - agreed!

**Pull request merged**